packages/contracts-bedrock/src/L1/SystemConfigInterop.sol

```diff
@@ -65,9 +65,9 @@ contract SystemConfigInterop is SystemConfig {
         Storage.setAddress(DEPENDENCY_MANAGER_SLOT, _dependencyManager);
     }
 
-    /// @custom:semver +interop
+    /// @custom:semver +interop-beta.0
     function version() public pure override returns (string memory) {
-        return string.concat(super.version(), "+interop");
+        return string.concat(super.version(), "+interop-beta.0");
```

Comment:
Note to reviewer: contracts-bedrock-checks fails without this version bump since the `SystemConfig` contract change and this contract (`SystemConfigInterop`) directly inherits `SystemConfig`.